test/fixtures/v2/did-revocation-ocsp-signed-revoked.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
hummm actually this document is only revoked because of the mock right? Technically it's not revoked

Reply:
yeap, but nothing in the document tells you its revoked, the only thing that tells u its revoked is a CRL/OCSP

Reply:
it's more that if I see this document name, I might reused it in another test and expect it to be revoked (and fail)

any reason to use two different documents for the test? Can use the same document and configure msw to fail in one case and not the other right?

Reply:
oh thats true actually, will change